DomainContent/FaceToFace/StageManager/stageManager.js

```diff
@@ -0,0 +1,208 @@
+//
+//  stageManager.js
+//
+//  Created by Rebecca Stankus on 06/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    var TABLET_BUTTON_IMAGE = Script.resolvePath('svg/megaphone-i.svg');
+    var TABLET_BUTTON_PRESSED = Script.resolvePath('svg/megaphone-a.svg');
+    var CLIP_1 = SoundCache.getSound(Script.resolvePath('sounds/audio1.wav'));
+    var CLIP_2 = SoundCache.getSound(Script.resolvePath('sounds/audio2.wav?12367'));
+    var SEARCH_RADIUS = 100;
+    var NUMBER_OF_MUTE_BALLS = 4;
+    var ONE_HUNDRED=100;
+
+    var audioVolume = 0.7;
+    var tablet = Tablet.getTablet('com.highfidelity.interface.tablet.system');
+    var appPage = Script.resolvePath('stageManager.html?482');
+    var screenSetupPage = Script.resolvePath('screenSetup.html?12');
+    var button = tablet.addButton({
+        text: 'STAGE',
+        icon: TABLET_BUTTON_IMAGE,
+        activeIcon: TABLET_BUTTON_PRESSED
+    });
+    var open = false;
+    var screens = [];
+    var backdrop;
+    var injector;
+    var silent = false;
+    var muteBalls = [];
+    var stageCenterPosition;
+    var ambisonicProperties = [];
+    var junglecube;
+    
+    function findTargets() {
+        Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element) {
+            var properties = Entities.getEntityProperties(element, ['name', 'position']);
+            if ((properties.name.indexOf("Today Web Entity Left") !== -1)) {
+                screens[0] = element;
+            } else if (properties.name.indexOf("Today Web Entity Right") !== -1) {
+                screens[1] = element;
+            } else if (properties.name.indexOf("Today Backdrop") !== -1) {
+                backdrop = element;
+            } else if (properties.name === "Red Carpet on Stage") {
+                stageCenterPosition = properties.position;
+            } else if (properties.name === "Jungle-Cube") {
+                junglecube = element;
+            }
+        });
+    }
+
+    function onClicked() { 
+        if (open) {
+            tablet.gotoHomeScreen();
+        } else {
+            tablet.gotoWebScreen(appPage);
+        }
+    }
+
+    function onWebEventReceived(event) {
+        if (typeof event === 'string') {
```

Comment:
This would be easier to read as a switch statement - also, ordered by most commonly used (I'd imagine that both screens might be more common than one or the other?)